### PR TITLE
cisco_ios_show_environment_temperature.template, iOS, show environment temperature

### PR DESCRIPTION
**Pull request by socretz**
##### ISSUE TYPE

 - Bugfix Pull Request

##### COMPONENT
cisco_ios_show_environment_temperature.template, iOS, show environment temperature

##### SUMMARY
Fixed requested changes in pull request #535 and tested against two stacked switches (9300-48UXM) provided in tests folder.  Also tested locally at GSK on 9500-40X. 

 
Fixes pull request 535
Fix 1:  Changed tox back to 36
Fix 2:  Added Error to each section as requested but this caused an error. Moved inlet temperature to the instance of "Inlet" and added error under each section.
Fix 3: Non-tabular fixed with Continue.Record
Fix 4: Added second switch to test file and tested changes to filter
Fix 5: \S+ used instead of \w
Fix 6: Removed Record from line 30 and just left start

 

 
Old
Start
  ^Switch\s+${SWITCH}:\s+SYSTEM\s+TEMPERATURE\s+is\s+${SWITCH_TEMPERATURE}\s*$$
  ^Inlet\s+Temperature\s+Value:\s+${INLET_TEMPERATURE_VALUE}\s+Degree\s+Celsius\s*$$ -> Inlet

Inlet
  ^Temperature\s+State:\s+${INLET_TEMPERATURE_STATE}\s*$$
  ^Yellow\s+Threshold\s+:\s+${INLET_YELLOW_THRESHOLD}\s+Degree\s+Celsius\s*$$
  ^Red\s+Threshold\s+:\s+${INLET_RED_THRESHOLD}\s+Degree\s+Celsius\s*$$ -> Hotspot

Hotspot
  ^Hotspot\s+Temperature\s+Value:\s+${HOTSPOT_TEMPERATURE_VALUE}\s+Degree\s+Celsius\s*$$
  ^Temperature\s+State:\s+${HOTSPOT_TEMPERATURE_STATE}\s*$$
  ^Yellow\s+Threshold\s+:\s+${HOTSPOT_YELLOW_THRESHOLD}\s+Degree\s+Celsius\s*$$
  ^Red\s+Threshold\s+:\s+${HOTSPOT_RED_THRESHOLD}\s+Degree\s+Celsius\s*$$ -> Record Start

New
Start
  ^Switch\s+\d -> Continue.Record
  ^Switch\s+${SWITCH}:\s+SYSTEM\s+TEMPERATURE\s+is\s+${SWITCH_TEMPERATURE}\s*$$ -> Inlet
  ^\s*$$
  ^. -> Error

Inlet
  ^Inlet\s+Temperature\s+Value:\s+${INLET_TEMPERATURE_VALUE}\s+Degree\s+Celsius\s*$$
  ^Temperature\s+State:\s+${INLET_TEMPERATURE_STATE}\s*$$
  ^Yellow\s+Threshold\s+:\s+${INLET_YELLOW_THRESHOLD}\s+Degree\s+Celsius\s*$$
  ^Red\s+Threshold\s+:\s+${INLET_RED_THRESHOLD}\s+Degree\s+Celsius\s*$$ -> Hotspot
  ^\s*$$
  ^. -> Error

Hotspot
  ^Hotspot\s+Temperature\s+Value:\s+${HOTSPOT_TEMPERATURE_VALUE}\s+Degree\s+Celsius\s*$$
  ^Temperature\s+State:\s+${HOTSPOT_TEMPERATURE_STATE}\s*$$
  ^Yellow\s+Threshold\s+:\s+${HOTSPOT_YELLOW_THRESHOLD}\s+Degree\s+Celsius\s*$$
  ^Red\s+Threshold\s+:\s+${HOTSPOT_RED_THRESHOLD}\s+Degree\s+Celsius\s*$$ -> Record Start
  ^\s*$$
  ^. -> Error

 
```

```

### CODE REVIEW

File: templates/cisco_ios_show_environment_temperature.template [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  ^Yellow\s+Threshold\s+:\s+${INLET_YELLOW_THRESHOLD}\s+Degree\s+Celsius\s*$$
+  ^Red\s+Threshold\s+:\s+${INLET_RED_THRESHOLD}\s+Degree\s+Celsius\s*$$ -> Hotspot
+  ^\s*$$
+  ^. -> Error 
+ 
+Hotspot
+  ^Hotspot\s+Temperature\s+Value:\s+${HOTSPOT_TEMPERATURE_VALUE}\s+Degree\s+Celsius\s*$$
+  ^Temperature\s+State:\s+${HOTSPOT_TEMPERATURE_STATE}\s*$$
+  ^Yellow\s+Threshold\s+:\s+${HOTSPOT_YELLOW_THRESHOLD}\s+Degree\s+Celsius\s*$$
+  ^Red\s+Threshold\s+:\s+${HOTSPOT_RED_THRESHOLD}\s+Degree\s+Celsius\s*$$ -> Record Start
```
Comment (FragmentedPacket) [inline templates/cisco_ios_show_environment_temperature.template]: You can remove the `Record` from the end of L30 and just leave **Start**

Comment (socretz) [reply templates/cisco_ios_show_environment_temperature.template]: Thank you.  Will do in morning and test.

Comment (socretz) [reply templates/cisco_ios_show_environment_temperature.template]: Thank you.  Will do in morning and test.

**Comment by socretz** [comment]
Added recommended changes.  See comments for details.

File: templates/cisco_ios_show_environment_temperature.template (hunk 1 above)
Comment (socretz) [reply templates/cisco_ios_show_environment_temperature.template]: wait.  I didn't push.  I am working on this now.  Sorry.  Give me about 30 minutes.

Comment (socretz) [reply templates/cisco_ios_show_environment_temperature.template]: Sorry for the delay.  It's fixed and pushed into the pull request

**Pull request merged**